### ISSUE TITLE
[V3][3.0.6] Error about privacy settings is thrown to console (Vimeo)

### ISSUE THREAD

**Issue by yuriykuzin** [sampotts/plyr#839]
Please check this codepen: https://codepen.io/yuriyku/pen/ZxaNPa

An error is thrown to console, but the video is still playing:

> Uncaught (in promise) Error: The URL is not available because of the video’s privacy settings.
>     at player.js:2
>     at Array.forEach (<anonymous>)
>     at y (player.js:2)
>     at u (player.js:2)

If I switch off Plyr (comment line `const player = new Plyr('#player');`), vimeo native player works properly.

### Expected behaviour

No error in console.

### Environment

- Browser: Chrome 
- Version: 65.0.3325.181
- Operating System: macOS Sierra
- Version: 10.12.6

### Steps to reproduce
Open the codepen and console

**Comment by sampotts** [comment]
I'd say that's probably a bug with Vimeo's API, player.js?

**Comment by yuriykuzin** [comment]
It could be. But there is no such error when I turn Plyr off and use only player.js, as I mentioned above.

**Comment by yuriykuzin** [comment]
@sampotts Can you please do a hotfix?

**File:** 

`src/js/plugins/vimeo.js`

**Change this:**
```
player.embed.getVideoUrl().then(value => {
            currentSrc = value;
        });
```
**To this:**
```
player.embed.getVideoUrl()
    .then(value => {
        currentSrc = value;
    })
    .catch(() => {
        // Do nothi'llng
    });
```

**Explanation:**

On a private video vimeo throws an error on calling `getVideoUrl`. And even with this catch, you will be still able to catch an error using `.on('error'...`, so it is quite safe

**Comment by sampotts** [comment]
Sure - will do this shortly

**Comment by yuriykuzin** [comment]
Thanks! I appreciate your fast responses! :)

**Comment by sampotts** [comment]
Fixed in `v3.0.8` 👍
